android/Staccato_AN/app/src/main/java/com/on/staccato/presentation/memoryupdate/viewmodel/MemoryUpdateViewModel.kt

```diff
@@ -169,6 +152,50 @@ class MemoryUpdateViewModel
             _isUpdateSuccess.setValue(true)
         }
 
+        private fun createThumbnail(
+            uri: Uri?,
+            url: String? = null,
+        ) {
+            val isEqualUri: Boolean? = _thumbnail.value?.isEqualUri(uri)
+            if (isEqualUri == false) {
+                thumbnailJobs[_thumbnail.value?.uri]?.cancel()
+                _thumbnail.value = ThumbnailUiModel(uri = uri, url = url)
+            }
+        }
+
```

Comment:
`thumbnailJobs[_thumbnail.value?.uri]?.cancel()` 로직을 deleteThumbnail() 내부로 옮기면 아래처럼 재사용 가능할 것 같네요!
```suggestion
            if (isEqualUri == false) {
                deleteThumbnail()
            }
            _thumbnail.value = ThumbnailUiModel(uri = uri)
```

Reply:
https://github.com/woowacourse-teams/2024-staccato/pull/542#discussion_r1837987150 와 동일한 의견입니다.